ground-station-openmct/terminal/terminal_logic.js

```diff
@@ -25,11 +25,11 @@ setInterval(function() {
 
 form.addEventListener("submit", function (event) {
     event.preventDefault();
-    const fullCommand = input.value;
-    let ack = fullCommand.match(/(\d+)/)[0];
-    ack = parseInt(ack)
-    const command = fullCommand.replace(/ .*/,'');
+    const fullCommand = input.value; //stores the user input in fullCommand
+    const commandParts = fullCommand.split(' ');
 
+    const command = commandParts[0]; // Extracts the command (first word)
+    const ack = parseInt(commandParts[1]); // Extracts the number (second word) and converts it to an int
     input.value = "";
 
     if (command === "clear") {
```

Comment:
Can you send the whole command to the backend and do this parsing in the views.py so that everytime we add a new command we don't have to edit two files?